packages/elements/src/canvas/__test__/canvas.test.js

```diff
@@ -70,7 +70,10 @@ describe('canvas/Canvas', () => {
     assert.strictEqual(el.ctx, el.getContext('2d'));
   });
 
-  it('Handles fractional pixelation', async () => {
+  it('Handles fractional pixelation', async function () {
```

Comment:
Why not arrow function ?

Reply:
We need to use `this.skip()` on the next line.